### ISSUE TITLE
On the stats dashboard, the "Where I Write" module is not displaying associated art.

### ISSUE THREAD

**Issue by timhkim** [nanowrimo/nanowrimo_frontend#274]
For example, here: https://preview.nanowrimo.org/stats for project "Draft 1", challenge "Draft 2".

If I wrote mostly at home, the correct art displays, but the art for "at the library", "at a cafe", "at the office", and "at [other]" is not displaying. Is the art available on the site, or do we need to upload it? If the latter, I can ask Simpatico for the SVGs. 

You can see the correct art in Zeplin here, as well: https://app.zeplin.io/project/5abe9c057d3a890444082509/screen/5abea027e57f6a03b8dceaf4

**Comment by timhkim** [comment]
@nanodave Could you possibly pull art for this and upload it here?

**Comment by nanodave** [comment]
Sent via email

**Comment by nanojezra** [comment]
ready for testing

**Comment by timhkim** [comment]
Hmmm. I think "at a café" doesn't seem to be working? I'm adding bulk counts to trigger seeing all the other images, which is working, but updates where my location is "at a café" don't even seem to be registering. Is it the special-charcter e?

**Comment by nanojezra** [comment]
Airbrake just recorded an error involving an attempt to create a new writing_location; which explains why the record failed, but "at a café" should already be a writing_location. 

How are you selecting "at a café"?

**Comment by timhkim** [comment]
I had to type it in! I was going to hang another ticket after this one, but I'm not seeing any pre-populated writing locations to select from in the updater when I click on that input field. I had to type them all in.

**Comment by nanojezra** [comment]
the locations are missing from the staging database

**Comment by nanojezra** [comment]
the error is indicative of submitting the same string ('at a café') multiple times. If 'at a café' is not being displayed as an option in the form, then there is yet another bug to report.

**Comment by timhkim** [comment]
Does it need the "é"? Like, if I put in "at a cafe", that won't register for the image?

**Comment by nanojezra** [comment]
correct. café needs to be spelled correctly; which it is not in the production DB, and the entry is missing from the staging DB.

**Comment by timhkim** [comment]
Ah, okay. Should I close this ticket and a make a more specific one for the cafe issue?

**Comment by nanojezra** [comment]
there are 2 issues at play.
1. the staging DB is not populated with NaNoWriMo's predetermined writing location strings 
2. a user entered string of "at a café" is failing to match "at a café" in the code

**Comment by timhkim** [comment]
Cool, so it sounds like we need to:

- [ ] Add predetermined writing locations to the staging DB.
- [ ] Settle on a "correct" input for the predetermined cafe location. 

For the latter, as long as it's set as a predetermined option, I'm fine with café. If I was trying to type it in on my own, I would never think to use the accent, but that's only a problem if it's not already pre-populated. We could also potentially go with "coffee shop" to get rid of any variable spellings, but again, if it's a predetermined option, this isn't an issue.

**Comment by timhkim** [comment]
Once the above two steps here are done, I think this is ready for production!
